src/content/overview/observability/logging/architecture/index.md

```diff
@@ -35,10 +38,21 @@ In this diagram, you can see that we run the following tools in each management
 
 - `Grafana Loki` that is accessible through our managed Grafana instance.
 - `multi-tenant-proxy`, a proxy component used to handle multi-tenancy for Loki.
-- A couple of logging agents (`Grafana Promtail` and `Grafana Agent`) that run on the management cluster and your workload clusters alike. We currently need two different tools for different purposes.
-    - Promtail is used to retrieve the container and kubernetes audit logs
+- A couple of scraping agents run on the management cluster and your workload clusters. There are different tools for different purposes:
+    - Promtail is used in older Giant Swarm releases to retrieve the container and kubernetes audit logs.
```

Comment:
```suggestion
    - Promtail is used in older Giant Swarm releases to retrieve the container and `Kubernetes` audit logs.
```

Reply:
IMO the code formatting is not correct in this context (same for other comment)

Reply:
```suggestion
    - Promtail is used in older Giant Swarm releases to retrieve the container and Kubernetes audit logs.
```